addons/website/static/src/builder/translation_components/selectTranslateDialog.js

```diff
@@ -12,20 +12,28 @@ export class SelectTranslateDialog extends Component {
         close: Function,
     };
     setup() {
-        this.inputEl = useRef("input");
+        this.inputEls = useRef("translateInputs");
+        this.selectTranslateEls = this.props.node
+            .querySelectorAll(".o_translation_select_option");
     }
 
-    onInputChange() {
-        const value = this.inputEl.el.value;
-        this.optionEl.textContent = value;
-        this.optionEl.classList.toggle(
-            "oe_translated",
-            value !== this.optionEl.dataset.initialTranslationValue
-        );
+    onInputChange(el) {
+        const inputs = Array.from(this.inputEls.el.querySelectorAll(".select-translate-input"));
```

Comment:
```suggestion
        const inputs = Array.from(this.inputEls.el.querySelectorAll("input"));
```

Won't this work? Why we need to introduce new class here?